pkg/splunk/enterprise/util.go

```diff
@@ -1443,22 +1441,33 @@ func checkIfFileExistsOnPod(cr splcommon.MetaObject, filePath string, podExecCli
 	return fileTestResult == 0
 }
 
+// resetStringReader resets the Stdin (of strings.Reader type) of the remotecommand.StreamOptions
```

Comment:
Is it more suitable to have `resetStringReader` part of  pkg/splunk/util/util.go?